company-profiles/stoneco.md

```diff
@@ -0,0 +1,31 @@
+# StoneCo
+
+## Company blurb
+
+[StoneCo](https://www.stone.co/) is a leading provider of financial technology solutions that empower merchants and integrated partners to conduct electronic commerce seamlessly across in-store, online, and mobile channels. Stone has developed a deep client-centric culture that seeks to delight our clients rather than to simply provide them with a solution or service. Stone serves clients of all sizes and types that transact online, offline or have an omni-channel sales approach. Stone also serves many integrated partners, which use or embed Stone’s solutions into their own offerings to enable their customers to conduct commerce more conveniently.
+
+Our purpose is clear: changing the way the payment system works. To do so, we work hard to provide an equal playing field for entrepreneurs.
+
+## Company size
+
+4000+ employees and constantly growing to innovate and bring solutions to our reason: the customer.
+
+## Remote status
+
+For many teams we offer full remote options, but you can work either in offices around Brazil as partially remote depending from your team and position.
```

Comment:
>you can work either in offices around Brazil as partially remote depending from your team and position.
This isn't very clear and there are a few errors with prepositions.
Maybe rephrase like this?
```suggestion
For many teams we offer fully remote options, and for the rest you can work either in offices around Brazil or partially remote depending on your team and position.
```